Branch/feature/workshop

jo-sch-01 opened the pull request:
pull workshop to master

a-kuntz committed:
xray client and server

a-kuntz committed:
workshop docu

a-kuntz committed:
add png

jo-sch-01 committed:
Put all parts together in workshop.md

Commit:
spelling

Commit:
fix compile issues in test, spelling

Commit:
remodel server diagram after issues

Commit:
delete get and set, return true on set Method

Commit:
delete unnecessary png files

Commit:
check if param is in range, interactive client

Commit:
fix parameter range

a-kuntz committed:
inherit dependency

jo-sch-01 committed:
Draft for user story

jo-sch-01 committed:
Added user stotys

Commit:
prioritize issues, add user story, spelling

a-kuntz commented on workshop/doc/workshop.md:
```diff
@@ -0,0 +1,237 @@
+## Helpfull ressources
+
+C++:
+* https://hackingcpp.com
```
please use markdown links. e.g.
```suggestion
* [hackingcpp](https://hackingcpp.com)
```

jo-sch-01 committed:
Changes from meeting

Commit:
remove unnecessary text, spelling, fix set methods

Commit:
fix wrong naming

jo-sch-01 committed:
Fixed test

Commit:
add plantuml diagrams as txt files

Commit:
link plantuml diagram to markdown

Commit:
link remaining plantuml diagrams

Commit:
reorder tests, fix spelling and other

antonio-mueller-itk committed:
Make small improvements to markdown files

* Fix typos
* Use action verbs to define goals and tasks
* Improve style consistency

antonio-mueller-itk committed:
Use bash to execute the bootstrap script

The bootstrap script uses the bash command `source`. The script cannot
be run using `sh`, it must be run using `bash`.

antonio-mueller-itk committed:
Instruct to clone the workshop branch

antonio-mueller-itk committed:
Include path to binaries in documentation

a-kuntz commented:
https://plantuml.com/de/server
